src/Test.FunctionalTests.BinSkim.Driver/BaselineTestData/Expected/Binskim.linux-x64.dll.sarif

```diff
@@ -210,10 +212,7 @@
                 "equivalentBinScopeRuleReadableName": "WXCheck"
               }
             }
-          ],
-          "properties": {
```

Comment:
>properties [](http://example.com/codeflow?start=11&length=10)

I will do you a favor and rebaseline main quickly. That will allow us to be better focused on your changes. Stand by and I will advice when you can merge from main.